tests/cpp/test_gpu_utils.cpp

```diff
@@ -46,9 +47,13 @@ TEST_F(NVFuserTest, FunctionTrace1) {
       ss.str(),
       ::testing::HasSubstr("Leaving myFavoriteFunction returning 3 at "));
   EXPECT_THAT(ss.str(), ::testing::HasSubstr("test_gpu_utils.cpp:31"));
+#else
+  GTEST_SKIP() << "Test only runs in debug mode";
```

Comment:
This part, where `NDEBUG` is not defined, should be for the debug mode, right?

Reply:
Hmm, yes, my mind was flipped...

Reply:
Changed `#ifndef NDEBUG` to `#ifdef NDEBUG`